1

```diff
@@ -0,0 +1,21 @@
+pick b08fd5c (SUP-2913) Puppet not updated for more than a year
```

Comment:
This file showed up again.

Reply:
gosh darn it! something i do in VI does that all the time.. i wish i knew what it was